cli/src/pcluster/validators/ec2_validators.py

```diff
@@ -189,27 +192,111 @@ def _validate(self, capacity_reservation_id: str, instance_type: str, subnet: st
                 )
 
 
+def get_capacity_reservations(capacity_reservation_resource_group_arn):
+    capacity_reservation_ids = AWSApi.instance().resource_groups.get_capacity_reservation_ids_from_group_resources(
+        capacity_reservation_resource_group_arn
+    )
+    print(capacity_reservation_ids)
```

Comment:
Let's use logger.info(...) if we want to record which capacity reservation IDs were found, otherwise we can simply avoid logging it in the first place.

Reply:
ty, I missed removing this print